docs/intro/intro_cqm.rst

```diff
@@ -13,7 +13,47 @@ typically formulate the constraints as penalties. Constrained models such as
 :class:`ConstrainedQuadraticModel` can support constraints by encoding both an
 objective and its set of constraints, as models or in symbolic form.
 
-CQM
-===
+Supported Models
+================
 
-TODO
+dimod provides a :term:`constrained quadratic model` (CQM) class that encodes
+a quadratic objective and one or more quadratic equality and inequality constraints.
+
+For an introduction to CQMs, see
+:std:doc:`Constrained Quadratic Models <oceandocs:concepts/cqm>`.
+
+For descriptions of the CQM class and its methods, see :ref:`cqm`.
+
+Model Construction
+==================
+
+dimod provides a variety of model generators. These are especially useful for testing
+code and learning.
+
+Example: dimod CQM Generator
+----------------------------
+
+This example creates a CQM representing a
+`knapsack problem <https://en.wikipedia.org/wiki/Knapsack_problem>`_` of ten
+items.
+
+>>> cqm = dimod.generators.knapsack(10)
+
+Typically you construct a model when reformulating your problem, using such
+techniques as those presented in D-Wave's system documentation's
+:std:doc:`oceandocs:doc_handbook`.
+
+Example: Formulating a CQM
+--------------------------
+
+This example constructs a CQM from symbolic math, which is especially useful for
+learning and testing with small CQMs.
+
+>>> x = BINARY('x')
+>>> y = INTEGER('y')
+>>> cqm = dimod.CQM()
+>>> objective = cqm.set_objective(x+y)
+>>> cqm.add_constraint(y <= 3)
+
+For very large BQMs, you might read the data from a file or construct from a NumPy
```

Comment:
Do you mean BQM or CQM here? I would probably use the generic "model".

Reply:
Yes, copy & paste error. Fixed